src/Conversion/InterventionImageFileConverter.php

```diff
@@ -90,16 +93,19 @@ private function validateOptions(array $options): array
     private function supportedByIntervention(string $format, InterventionBackend $backend): bool
     {
         $driver = $backend->getImageManager()->config['driver'] ?? null;
-        // If the driver is somehow not GD or Imagick, we have no way to know what it might support
-        if ($driver !== 'gd' && $driver !== 'imagick') {
-            return false;
-        }
 
         // Return early for empty values - we obviously can't support that
         if ($format === '') {
             return false;
         }
 
+        // If the driver is somehow not GD or Imagick, we have no way to know what it might support
+        if ($driver !== 'gd' && $driver !== 'imagick') {
+            $supported = false;
+            $this->extend('updateSupportedByIntervention', $supported, $driver);
+            return $supported;
+        }
```

Comment:
Note that there's no legitimate use case for updating the value if the driver is gd or imagick, since the logic below exactly mirrors the logic those drivers use internally.